dash_app/dashboard.py

```diff
@@ -87,22 +87,30 @@ def init(server):
     )
 
     counties = ds.get_subset(aggregation_level=AggregationLevel.COUNTY)
-    county_stats = timeseries_stats.PerVariable.make(counties)
+    county_stats = timeseries_stats.PerTimeseries.make(counties)
     county_variable_population_ratio = pd.DataFrame(
         {
             "has_url": population_ratio_by_variable(counties, county_stats.has_url),
             "has_timeseries": population_ratio_by_variable(counties, county_stats.has_timeseries),
         }
     )
 
-    region_df = region_table(per_region_stats_all_vars, ds)
+    region_df = region_table(per_timeseries_stats, ds)
 
     dash_app.layout = html.Div(
         children=[
             html.H1(children="CAN Data Pipeline Dashboard"),
             html.P(commit_str),
             html.H2("Time-series count"),
+            html.H3("By variable group"),
             dash_table_from_data_frame(agg_stats.has_timeseries, id="agg_has_timeseries"),
+            html.H3("By demographic distribution"),
+            dash_table_from_data_frame(
+                per_timeseries_stats.aggregate(
```

Comment:
nit: pull out the `per_timeseries.aggregate()` into it's own variable to save a line and makes it a bit more clear where the work is being done.  Right now I have to keep a bit more context on my mental stack when reading everything that's going into the first input for `dash_tabvle_from_data_frame`

Reply:
Done.